connector_amazon/models/order.py

```
@@ -0,0 +1,777 @@
+            ET.SubElement(fulfillment_data, "CarrierName").text = carrier_name
+            if tracking_ref:
+                ET.SubElement(fulfillment_data, "ShippingMethod").text = tracking_ref
+                ET.SubElement(
+                    fulfillment_data, "ShipperTrackingNumber"
+                ).text = tracking_ref
+
+        # Add line items (shipped quantities)
+        for move in picking.move_ids.filtered(lambda m: m.state == "done"):
+            # Try to find corresponding order line to get Amazon item ID
+            order_line = self.odoo_id.order_line.filtered(
+                lambda l: l.product_id == move.product_id
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)